modules/navier_stokes/doc/content/source/functormaterials/FunctorErgunDragCoefficients.md

```diff
@@ -0,0 +1,24 @@
+# FunctorErgunDragCoefficients
+
+!syntax description /Materials/FunctorErgunDragCoefficients
+
+This class implements the Darcy and Forchheimer coefficients for the Ergun drag
+force model which is discussed in detail in
+[PINSFVMomentumFriction.md#friction_example]. To summarize, this class implements
```

Comment:
I would also mention "see the kernel for the definition of the friction coefficients"